### PR TITLE
Cognito pool env var

### PR DESCRIPTION
**Pull request by MancunianSam**
The name of this branch is wrong because I realised after I started that all of the stage specific webpack variables needed to be replaced.

### CODE REVIEW

File: app/configuration/FrontEndInfoConfiguration.scala [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package configuration
+
+import javax.inject.Inject
+import play.api.Configuration
+import viewsapi.FrontEndInfo
+
+class FrontEndInfoConfiguration @Inject ()(configuration: Configuration) {
```
Comment (TomJKing) [inline app/configuration/FrontEndInfoConfiguration.scala:7]: Wonder whether "...Info..." is the best word to use in the name? Seems a bit vague. Although can't really think of a better one.

File: conf/application.conf [hunk 2]
```diff
@@ -6,12 +6,19 @@ auth.url=${?AUTH_URL}
 auth.callback="http://localhost:9000/callback"
 auth.secret="placeholderSecretValue"
 auth.secret=${?AUTH_SECRET}
+
+cognito.identitypool=${?IDENTITY_POOL_ID}
+cognito.identityProviderName=auth.tdr-integration.nationalarchives.gov.uk/auth/realms/tdr
```
[the remark this answers is not in the document]
Comment (MancunianSam) [reply conf/application.conf:11]: No, local keycloak won't work and I don' think there's a way of setting up an iam identity provider to connect to a local server which is upsetting.

File: app/configuration/FrontEndInfoConfiguration.scala (hunk 1 above)
Comment (MancunianSam) [reply app/configuration/FrontEndInfoConfiguration.scala:7]: I'm not too pleased either, it was originally authenticationDetails or something like that but there's the api url and the stage so I tried to go for something generic.

File: conf/application.conf (hunk 2 above)
[the remark this answers is not in the document]
Comment (TomJKing) [reply conf/application.conf:11]: Makes sense

File: app/configuration/FrontEndInfoConfiguration.scala (hunk 1 above)
Comment (TomJKing) [reply app/configuration/FrontEndInfoConfiguration.scala:7]: Fair enough

**Pull request merged**